spec/features/discovery_report/completed_with_errors_spec.rb

```diff
@@ -41,7 +41,7 @@
     # go to job details page, wait for preassembly to finish
     first('td  > a').click
     expect(page).to have_content project_name
-    expect(page).to have_content 'Discovery report completed (with errors)'
+    expect(page).to have_content 'Discovery report completed (with preassembly errors)'
```

Comment:
this seems like a weird bit of text to have for the discovery report.  could `JobRun#human_state_name` either do something more generic for the parenthetical, or pick `discovery report` or `preassembly` for the text based on the job type?  i actually might vote for the former, since it's simpler and surrounding context should fill in that info fine?  "Discovery report completed (with errors)" seems sufficiently informative?

Reply:
Yes. I don't think it's possible to have preassembly errors in a discovery report. The discovery report runs on its own.

Reply:
https://github.com/sul-dlss/pre-assembly/issues/1316